pkg/master/master.go

```diff
@@ -94,7 +94,7 @@ const (
 	// DefaultEndpointReconcilerTTL is the default TTL timeout for the storage layer
 	DefaultEndpointReconcilerTTL = 15 * time.Second
 	// DefaultStorageEndpoint is the default storage endpoint for the lease controller
-	DefaultStorageEndpoint = "kube-apiserver-endpoint"
+	DefaultStorageEndpoint = "services/endpoints"
```

Comment:
@deads2k @sttts @liggitt is there any better way to get at this value?

Reply:
we shouldn't need this at all. this is getting passed like this:
```
ResourcePrefix:          c.ExtraConfig.StorageFactory.ResourcePrefix(kapi.Resource(DefaultStorageEndpoint)),
```

that should be 
```
ResourcePrefix:          c.ExtraConfig.StorageFactory.ResourcePrefix(kapi.Resource("endpoints")),
```

Reply:
Partial diff [here](https://github.com/kubernetes/kubernetes/compare/master...rphillips:fixes/lease_endpoint_fix)

Reply:
ah, the api.Resource("endpoints") is even better

Reply:
I think we can get it from StorageFactory.ResourcePrefix(api.Resource("endpoints"))